### PR TITLE
Made slot game feature possible at testnet

### PR DESCRIPTION
**Pull request by fish-sushi**

### CODE REVIEW

File: app/javascripts/helpers/web3Service.jsx [hunk 1]
```diff
@@ -1,41 +1,78 @@
 import Web3 from 'web3';
-import contract from 'truffle-contract';
 import EnvChecker from './envChecker';
+const mockABI = require('./__mocks__/mockABI.json');
+
+const CONTRACT_ADDRESS = '0x6225df7d671e0278b5bc32144fe88f5139edba0a';
```
Comment (parkkoon) [inline app/javascripts/helpers/web3Service.jsx:5]: Isn't it better to replace this static address as another thing?

**Review by parkkoon — APPROVED** [review]
👍

File: app/javascripts/helpers/web3Service.jsx (hunk 1 above)
Comment (fish-sushi) [reply app/javascripts/helpers/web3Service.jsx:5]: This address can be changed later.

**Review by c0smic0wl — APPROVED** [review]
🦉

**Pull request merged**